common/sched.c

```diff
@@ -226,7 +247,7 @@ void run_tasks(cpu_t *cpu) {
                 printk("%s task '%s' finished on CPU[%u] with result %ld\n",
```

Comment:
I suppose a task does not finish until all repetitions are through? Otherwise, maybe a summary of how many times the task finished. For short task this and the `"CPU[1]: Running task ...`" messages get a bit spammy

Reply:
Alright, I will try to improve that. Will simply print summary after all repetitions are through.